plugins/admission/Dockerfile

```diff
@@ -0,0 +1,28 @@
+FROM golang:1.14.3 AS build-env
+WORKDIR /go/src/github.com/fairwindsops/insights-plugins/
+
+
+ENV GO111MODULE=on
+ENV CGO_ENABLED=0
+ENV GOOS=linux
+ENV GOARCH=amd64
+
+COPY opa opa
```

Comment:
should we move this down below the `copy go.*`? That way if we change the `opa` code the deps are still cached.

Reply:
Unfortunately because OPA is one of the dependencies it needs to be there before the go get.